lib/fetcher.js

```diff
@@ -1,4 +1,3 @@
-const fetch = require("node-fetch");
```

Comment:
Removing node-fetch will fail on Node.js instances because fetch is not defined.
![image](https://user-images.githubusercontent.com/26350892/210432277-f4fd1e0f-44cb-443f-810c-567a0d8de486.png)

Reply:
I've implemented `fetch` on recent projects, and I didn't need to usage node-fetch dependency.

Reply:
It is the version of Node. I switched from v16 to v18 and it worked just fine. 

For the time being, I think we should still have node-fetch in for v16 and under compatibility since v14 and v16 are still in maintenance mode for most of this year and v16 was the one with this issue.